lib/service.js

```diff
@@ -17,6 +17,44 @@
 const rclnodejs = require('bindings')('rclnodejs');
 const Entity = require('./entity.js');
 const debug = require('debug')('rclnodejs:service');
+const {MessageTranslator} = require('./message_translator.js');
+
+/**
+ * The response to client
+ *
+ * @property {Object} template - Get an empty response object that is being sent to client.
+ * @property {Service} service - The service that this response object is attaching to.
+ *
+ * @hideconstructor
+ */
+class Response {
+  constructor(service, header) {
+    this._service = service;
+    this._header = header;
+    this.sent = false;
+  }
+
+  get template() {
+    return MessageTranslator.toPlainObject(new this._service._typeClass.Response());
+  }
+
+  get service() {
+    return this._service;
+  }
+
+  /**
+   * Send response to client (the service caller)
+   * @param {object} response - The plain JavaScript representing the response.
+        Note: you can use .template to get an empty result object.
+   * @return {undefined}
+   * @see {@link Response#template}
+   */
+  send(response) {
+    const rawROSResponse = this._service.translator.toROSResponse(response).serialize();
+    rclnodejs.sendResponse(this._service._handle, rawROSResponse, this._header);
+    this.sent = true;
+  }
+}
```

Comment:
I didn't see the corresponding class, something like `class Request`, in `lib/client.js`, is it necessary here?